.changeset/odd-kangaroos-give.md

```diff
@@ -0,0 +1,6 @@
+---
+"ledger-live-desktop": patch
+"live-mobile": patch
```

Comment:
Just we should still separate LLM and LLD changeset i would say

Reply:
We don't need to anymore as we version both at the same time now